app/whats-new/whats-new.njk

```diff
@@ -1,19 +1,35 @@
 {% extends "layouts/base.njk" %}
 
 {% block pageTitle %}
-  {{ heading }} – {{ super() }}
+  What's new
 {% endblock %}
 
-{% block content %}  
-  {% if contentBody and contentTitle %}
-    <header>
-      <h1 class="govuk-heading-xl govuk-!-margin-top-1 govuk-!-margin-bottom-6">
-        {{ contentTitle }}
-      </h1>
-    </header>
+{% block content %}
+  <header>
+    <h1 class="govuk-heading-xl govuk-!-margin-top-1 govuk-!-margin-bottom-6">
+      What's new
+    </h1>
+  </header>
+
+  <p class="govuk-!-margin-bottom-5">
+    The latest updates and improvements to Book a Secure Move.
+  </p>
+
+  {% for post in posts %}
+    {% if post.fields.body and post.fields.title %}
+      <h2 class="govuk-heading-m govuk-!-margin-top-4 govuk-!-margin-bottom-5">
+        {{ post.fields.date }}
+      </h2>
+
+      <hr>
+      
+      <h3 class="govuk-heading-l govuk-!-margin-top-5">
+        {{ post.fields.title }}
+      </h3>
+        
+      {{ post.fields.body | safe }}
+      {% else %}
```

Comment:
This might be a daft question but what purpose does the `{% else % }` serve?   i.e. is it needed?

Reply:
Was going to originally have a message if we had no content to show, but think thats changed, so the else is not needed now